spec/controllers/idv/session_errors_controller_spec.rb

```diff
@@ -122,6 +122,7 @@
   before do
     allow(idv_session).to receive(:verify_info_step_complete?).
       and_return(verify_info_step_complete)
+    allow(idv_session).to receive(:address_verification_mechanism).and_return(nil)
```

Comment:
I think we need to avoid this since `idv_session` can be nil and won't be able to receive the method